packages/react/src/index.ts

```diff
@@ -1,21 +1,4 @@
-// Fix Safari's nonexistent browser.storage https://github.com/chainapsis/keplr-wallet/blob/4726a96b9663f17b91c5d6b0448bf85ebb4a678a/packages/common/src/kv-store/extension.ts
-
-// eslint-disable-next-line @typescript-eslint/no-namespace
-declare namespace browser.storage {
-  const local: {
-    get: undefined
-    set: undefined
-  }
-}
-
-if (
-  typeof window !== 'undefined' &&
-  typeof browser !== 'undefined' &&
-  typeof browser.storage === 'undefined'
-) {
-  browser.storage = { local: { get: undefined, set: undefined } }
-}
-
 export * from './components'
 export * from './types'
-export * from './utils'
+
+export * from '@cosmos-wallet/core'
```

Comment:
hmm... do we want to do this? i think this might relate to whether we want to bundle core with the react app or not?

Reply:
Yeah, I did this for now because it made changing the example / docs simple, but it might not be a bad idea to change the API and make them import from core. I think it makes sense not to bundle them together.